<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGRenameSceneDlg</class>
 <widget class="QDialog" name="GGRenameSceneDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename scene</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Scene Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSceneName"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRenameMediaDir">
     <property name="text">
      <string>Also rename media folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtMediaDir">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtSceneName</tabstop>
  <tabstop>chkRenameMediaDir</tabstop>
  <tabstop>txtMediaDir</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGRenameSceneDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGRenameSceneDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRenameMediaDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtMediaDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSceneName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GGRenameSceneDlg</receiver>
   <slot>sceneNameEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sceneNameEdited(QString)</slot>
 </slots>
</ui>
